.github/renovate.json5

```diff
@@ -21,6 +21,11 @@
       "matchPackagePrefixes": ["org.eclipse.jetty:"],
       "groupName": "jetty packages"
     },
+    {
+      // OpenTelemetry SDK updates are handled by auto-update-otel-sdk.yml
+      "matchPackagePrefixes": ["io.opentelemetry:"],
```

Comment:
Do you think we could remove the `"io.opentelemetry:opentelemetry-bom"` rule now?